frontend/src/pages/CourseEditor/DefinitionStep/styled.ts

```diff
@@ -10,21 +14,23 @@ export const FieldWrapper = styled(Box)({
   },
 });
 
-export const Field = styled(TextField)({
-  width: '350px',
-  height: '53px',
-  fontSize: '18px',
-  '& label': {
-    color: '#A2A2A2',
-  },
-  '&.MuiTextField-root > .MuiFormLabel-root.Mui-focused': {
-    color: '#2C2525',
-  },
-  '& .MuiFormHelperText-root': {
-    marginRight: '29px',
-    marginBottom: '5px',
-  },
-});
+export const Field = styled(TextField)<ICustomInput>(({ isLinkType }) => ({
+  ...(isLinkType && {
+    width: '350px',
+    height: '53px',
```

Comment:
try to avoid odd values
https://habr.com/ru/post/252395/
https://habr.com/ru/company/productivity_inside/blog/319700/